frontend/src/i18n/en/index.ts

```diff
@@ -6,7 +7,7 @@ const translation = {
       },
     },
     app: {
-      name: 'Bedrock Claude Chat',
+      name: !ENABLE_MISTRAL ? 'Bedrock Claude Chat' : 'Bedrock Chat',
```

Comment:
Please declare `nameWithoutClaude` and leave the current `name` so that we don't have to maintain when new language is added.

Reply:
So does it mean you prefer to keep all i18n fields intact, but add logic from the frontend `.ts` code?

something like
```javascript
const ENABLE_MISTRAL: boolean = import.meta.env.VITE_APP_ENABLE_MISTRAL === 'true';

...
document.title = !ENABLE_MISTRAL? t('app.name'): 'Bedrock Chat';
...
```

Reply:
Yes, because it enables us not need to add any changes to the i18n translation files.
```ts
const ENABLE_MISTRAL: boolean = import.meta.env.VITE_APP_ENABLE_MISTRAL === 'true';

...
document.title = !ENABLE_MISTRAL? t('app.name'): ('app.nameWithoutClaude');
...
```

@wadabee 
How do you think?

Reply:
Sorry for the late reply.
I agree to add `nameWithoutClaude` to the i18n file.
This is because I don't want to hard-code text within the TSX, and I don't want to write logic in the i18n file.